### PR TITLE
[Integrations] Updated the infrastructure category

### LINKED ISSUE #4469: Tag few integrations under the new `infrastructure` category
Please tag the below list of integration under the new category `infrastructure`.

| Packages | 
| ---------  |
| activemq  | 
| apache |
| couchbase| 
| couchdb| 
| etcd| 
| hadoop |  
| haproxy | 
| http_endpoint | 
|ibmmq|
| websphere_application_server |
| iis |
| kafka |
| memcached |
| mongodb |
| microsoft_sqlserver |
| mysql |
| nagios_xi |
| nginx |
| oracle |
| postgresql |
| rabbitmq |
| redis |
| redisenterprise |
| traefik |
| vsphere |
| zookeeper |

References: https://docs.google.com/document/d/1Ha-rqDbMAYIt8KIU-E6dZqU4WyQ0AjbeiY3lKxeP7nc/edit

Github issue associated: https://github.com/elastic/package-spec/issues/381

### PR DESCRIPTION
**Pull request by ManojS-shetty**
- Enhancement

## What does this PR do?

This PR tags the infrastructure integrations under a new category called as `infrastructure` .

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally
- Clone the PR branch to local.
- Run the command to bring up the stack `elastic-package stack up`
- Navigate to Browse integrations page.
- Select the elastic-agent only option on the page.
- In the left category panel click on the infrastructure category.
- This should result in listing all the integration which is tagged under this new category.

## Related issues

- Closes #4469 

CC: @rameshelastic @lalit-satapathy

### CODE REVIEW

**Review by efd6 — APPROVED** [review]
Changes for http_endpoint and oracle LGTM.

File: packages/couchdb/manifest.yml [hunk 1]
```diff
@@ -1,13 +1,14 @@
 format_version: 1.0.0
 name: couchdb
 title: CouchDB
-version: 0.1.0
+version: 0.2.0
```
[the remark this answers is not in the document]
Comment (ManojS-shetty) [reply packages/couchdb/manifest.yml]: Done . maintained everything x.y.**Z**

[the remark this answers is not in the document]
Comment (ManojS-shetty) [reply packages/couchdb/manifest.yml:4]: Done

**Pull request merged**